wagtailio/static/sass/components/_sub-nav.scss

```diff
@@ -0,0 +1,214 @@
+.sub-nav {
+    @include z-index(nav);
+    $root: &;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    width: 100%;
+    height: calc(100% - #{$header-mobile-height});
+    position: absolute;
+    list-style: none;
+    top: 0;
+    left: 0;
```

Comment:
Thought: surprised the linter didn't moan about using `inset` here

Reply:
updated all left to inset for consistency